broker-daemon/broker-rpc/admin-service/register.js

```diff
@@ -15,8 +23,8 @@ async function register ({ relayer, logger }, { RegisterResponse }) {
   const { entityId } = await relayer.adminService.register({ publicKey })
 
   logger.info('Successfully registered public key with relayer', {entityId})
-
-  return new RegisterResponse({ entityId })
+  const url = `${REGISTER_URL}/${entityId}`
```

Comment:
[nit] I think the slash should probably be part of the constant, or you should do a `join` with the slash.